cmd/cel-key/node_types.go

```diff
@@ -54,7 +55,11 @@ func ParseDirectoryFlags(cmd *cobra.Command) error {
 	}
 	switch nodeType {
 	case "bridge", "full", "light":
-		keyPath := fmt.Sprintf("~/.celestia-%s-%s/keys", nodeType, strings.ToLower(network))
+		home, err := os.UserHomeDir()
+		if err != nil {
+			return err
+		}
+		keyPath := fmt.Sprintf("%s/.celestia-%s-%s/keys", home, nodeType, strings.ToLower(network))
```

Comment:
can we have helper func, somewhere in the root of cmd, so I don't need to change this path 3 times

Reply:
done https://github.com/celestiaorg/celestia-node/pull/1565/commits/78224d01c6f4fa42224027a134ea4b17af91e4f2